### ISSUE TITLE
ability to copy the characters in the "missing" row

### ISSUE THREAD

**Issue by jpt** [justinpenner/TalkingLeaves#6]
it would be useful to be able to copy those glyphs to, for example, research / look more deeply into a character / ductus / particular orthography.  a workaround is for me to just add the glyphs and then copy them, but I probably want to look into it first so I would prefer not to

**Comment by justinpenner** [comment]
This is something I was thinking about, too. I think it would be nice to have a right-click context menu in the Languages list, which would show several functions like:

- Look up [language name] on Wikipedia _when only 1 language is selected_
- Look up missing character on Wikipedia
  - [missingChar1]
  - [missingChar2] _etc_
- Copy selection as CSV
- Copy entire table as CSV
- Copy missing characters
  - Space Separated
  - One Per Line
  - Python List
- Copy missing glyph names
  - Space Separated
  - One Per Line
  - Python List
- Copy missing codepoints
  - Space Separated
  - One Per Line
  - Python List

**Comment by jpt** [comment]
that's far more comprehensive than what I had in mind - a context menu with these options would be very useful (though I can't imagine using the CSV function, personally)

**Comment by justinpenner** [comment]
I was thinking some users may want to copy rows from the languages table and paste them into an app that supports tables or spreadsheets. It might make more sense to call them _Copy selected rows_ and _Copy all rows_, then put them in clipboard as tab-separated values, which most spreadsheet apps should be able to handle via paste.

**Comment by schriftgestalt** [comment]
You should be able to make the text cell selectable and then on can pick what to copy paste.

**Comment by justinpenner** [comment]
> You should be able to make the text cell selectable and then on can pick what to copy paste.

I tried this but Vanilla doesn't seem to have an option for it. Cells in a List2 table can be editable, but not selectable. The cells are implemented as NSTextField, which can be selectable without being editable, but I tried that, and it doesn't seem to work when it's inside a table.

I might come back to it, but I have to be careful with anything that requires hacking out of Vanilla to pyobjc, because I'm not familiar with Apple's frameworks and pyobjc is made for developers who are, so any small thing can take a long time to figure out.

For now, I'm working on the context menu I mentioned above, so that will have an option to copy the whole row to clipboard, including the missing characters.
